install/css/install.css

```diff
@@ -191,6 +191,10 @@ input[type="url"] {
 	width: 100%;
 }
 
+input[type="number"] {
```

Comment:
Ignore my comment. As we're already styling bare elements like `input`, we can continue.

Ideally I prefer a framework to only style elements with prefixed classnames so plugins can add their own markup and style it as necessary without interference. But we're well beyond that.